### PR TITLE
e2e:chore - tests for the output format and json outuput file flags

### PR DESCRIPTION
**Pull request by lucasbrunozup**
Add tests for the output format and json outuput file flags
Signed-off-by: lucas.bruno <lucas.bruno@zup.com.br>

### CODE REVIEW

File: e2e/commands/start/start_test.go [hunk 1]
```diff
@@ -18,36 +18,33 @@ import (
...
-		flags             map[string]string
-		repoAuthorization string
-		configFilePath    = testutil.GoExample1
+		session        *gexec.Session
+		flags          map[string]string
+		configFilePath = testutil.GoExample1
 	)
 
 	JustBeforeEach(func() {
+		var err error
```
Comment (iancardosozup) [inline e2e/commands/start/start_test.go:42]: this error can't be started in `session, err := gexec.Start(cmd, GinkgoWriter, GinkgoWriter)`?

File: e2e/commands/start/start_test.go [hunk 2]
```diff
@@ -132,4 +130,22 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 			Expect(session.Out.Contents()).To(ContainSubstring(fmt.Sprintf(`\"repository_authorization\": \"%s\"`, testutil.NormalizePathToAssertInJSON(repoAuthorization))))
 		})
 	})
+
+	When("--output-format and --json-output-file is passed", func() {
```
Comment (iancardosozup) [inline e2e/commands/start/start_test.go:134]: Could add the others output types besides json ? i think it's sonarqube and a invalid type.

Comment (lucasbrunozup) [reply e2e/commands/start/start_test.go]: Added txt and sonoarqube testecases.

File: e2e/commands/start/start_test.go (hunk 1 above)
Comment (lucasbrunozup) [reply e2e/commands/start/start_test.go:42]: do i need the session to be "global", any suggestion?

**Review by iancardosozup — APPROVED** [review]

**Review by nathanmartinszup — APPROVED** [review]

**Pull request merged**